case_studies/san_leandro_high_school/01_club_fair/planning.md

```diff
@@ -1 +1,22 @@
 # Planning
+
+##### Plans for Event:
+  1. Club table
+    * Hack club banner/poster
+    * Explaining our mission/what we do.
+  2. Sign-ups
+    * 2 sheets
+    * Name, email, phone #
+    * Will be transferred to Google Doc afterward
+  3. Flyers to pass out
+    * Approximately 4 in * 4 in
+    * Logo, meeting times, contact info
```

Comment:
Couple points to make this conform to our Markdown guidelines:

- Remove indentation
- Use `-` for bullet points instead of `*`
- Add a blank new line after line 3